tests/test_deployment.py

```diff
@@ -2,7 +2,7 @@
 import numpy.typing as npt
 import pytest
 import torch
-from random_data import make_feature_file
+from random_data import make_old_feature_file
```

Comment:
I think it would be better to instead use the new `make_feature_file`, or (even better) parameterise the tests as mentioned above.